backend/src/crd/controller/scheduledworkflow/main.go

```diff
@@ -102,6 +113,7 @@ func initEnv() {
 func init() {
 	initEnv()
 
+	flag.StringVar(&logLevel, "", "Defines the log level for the application.")
```

Comment:
Seems like you missed an argument according to the e2e test. I would recommend you try to run the docker image on your local machine to catch the start up errors so you don't have to wait for the e2e test to catch each bug.

```suggestion
	flag.StringVar(&logLevel, "logLevel", "", "Defines the log level for the application.")
```

Reply:
Apologies, my bad. I had this change saved locally but it didn't get git push'ed. Updated now and the tests have passed as well. :)